code/index.html

```diff
@@ -4,9 +4,190 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="./style.css">
     <title>Project 4: Pizzabot</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&family=Staatliches&display=swap&family=Montserrat&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>This is your Pizza Bot</h1>
+    <section class="flex-wrapper">
+      <h1>Welcome to the Pizza Bot</h1>
+      <!-- Form -->
+      <section class="flex-container visible" id="flex-container">
+      <form 
+        action="" 
+        id="order-form" 
+        name="order-form">
+
+        <!-- Chat window 1 class="flex-container visible"-->
+        <div 
+          class="input-wrapper visible" 
+          id="chatWindow1">
+          <p>
+            Hey! Happy to serve your pizza. Please enter your name
+          </p>
+          <label 
+            for="name"
+            class="label">
+            Name:
+          </label>
+          <input 
+            type="text" 
+            class="input" 
+            name="name" 
+            id="name">
+          <button 
+            onclick="clickNextButton(this.parentNode.id)" 
+            type="button" 
+            class="button">
+            Continue
+          </button>
+        </div>
+
+        <!-- Chat window 2 -->
+        <div 
+          class="input-wrapper invisible" 
+          id="chatWindow2">
+          <p 
+            id="order">
+          </p>
+          <div class="custom-select">
+          <select 
+            name="pizza" 
+            class="input" 
+            id="pizza">
+            <option 
+              value="">
+              Choose a pizza
+            </option>
+            <option 
+              value="vegetarian">
+              Vegetarian Pizza
+            </option>
+            <option 
+              value="hawaiian">
+              Hawaiian Pizza
+            </option>
+            <option 
+              value="pepperoni">
+              Pepperoni Pizza
+            </option>
+            <option 
+              value="capricciosa">
+              Capricciosa Pizza
+            </option>
+            <option 
+              value="margherita">
+              Margherita Pizza
+            </option>
+          </select>
+          </div>
+          <button 
+            onclick="clickNextButton(this.parentNode.id)" 
+            type="button" 
```

Comment:
There is one thing going on over here : because you've added function to `onclick` event + set `type="button"`, your application works well only when clicking on button. I tried to submit your form by clicking `enter` key and whole app stops working.
This leads us to one important lesson - we should always use `onsubmit` event attached to `form` + `<button type="submit" />` rather than attaching `onclick` to `button`. First case will cover both submitting form by hiting `enter` and clicking button, second will cover only button-click case